base/applications/cmdutils/donutils.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * PROJECT:     ReactOS command-line utilities
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Fallback of <conutils.h>
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+#pragma once
```

Comment:
What's the purpose of this header?

Reply:
Providing functionality outside of ReactOS.

Reply:
For example:
https://github.com/katahiromz/where

Reply:
`donutils.h` is renamed as `miniconutils.h`.